classes/module/Module.php

```diff
@@ -758,7 +758,7 @@ public function enable($force_all = false)
             }
         }
 
-        if ($this->getOverrides() != null) {
+        if ($force_all && $this->getOverrides() != null) {
```

Comment:
Hi @eternoendless I know, but the default behavior is not correct as well because if you disable the module only on one shop you will lose the overrides for the other shop.
I'm trying to edit my solution in order to take the count the existing shops and compare it with the getContextListShopID. If they matches we can uninstall the overrides otherwise they should not be uninstalled.